pkg/apis/apps/v1/zz_generated.defaults.go

```diff
@@ -109,7 +107,7 @@ func SetObjectDefaults_DaemonSet(in *v1.DaemonSet) {
 		corev1.SetDefaults_Container(a)
 		for j := range a.Ports {
 			b := &a.Ports[j]
-			if reflect.ValueOf(b.Protocol).IsZero() {
```

Comment:
the `zz_generated.defaults.go` changes are unexpected and were not in the master PR